decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -105,19 +107,22 @@
             </div>
           </div>
         <% end %>
-        <div class="card-section">
-          <div class="row column">
-            <span class="component__show-title"><%= t ".related_meetings" %></span>
-            <ul id="related-meetings">
-              <% proposal.linked_resources(:meetings, "proposals_from_meeting").each do |meeting| %>
-                <% presented_meeting = present(meeting) %>
-                <li>
-                  <%= link_to presented_meeting.title(html_escape: true), presented_meeting.profile_path %>
-                </li>
-              <% end %>
-            </ul>
+        <% proposal_meetings = proposal.linked_resources(:meetings, "proposals_from_meeting") %>
+        <% if proposal_meetings.any? %>
+          <div class="card-section">
```

Comment:
I am thinking maybe is worth extracting to a partial ?

Reply:
Same here, I propose to do it as another PR

Reply:
works for me.